examples/android/src/main/java/com/dropbox/core/examples/android/DropboxActivity.java

```diff
@@ -12,20 +15,43 @@
  */
 public abstract class DropboxActivity extends AppCompatActivity {
 
+    public final static boolean USE_SLT = false; //If USE_SLT is set to true, our Android example will use our beta feature Short Live Token.
```

Comment:
Is there any other way to structure this?  I don't love having a public global flag to control this.  What if we used shared preferences?

Reply:
Since this is just for example purpose and should be eventually removed. I would like just to change it to private